ENVIRONMENT.md

```diff
@@ -25,13 +25,15 @@
    ```env
    DISCORD_TOKEN = <your discord bot token>
    STAFF_CHANNEL = <the channel ID where new applications are sent>
+   STAFF_ROLE = <the role ID for staff members>
```

Comment:
nit: I would prefer this to be named `ALLOW_SURVEY_ROLE_ID` to maintain the same naming convention as `ALLOW_SURVEY_CHANNEL_ID`